languages/kotlin/config.json

```diff
@@ -3,7 +3,8 @@
   "language": "Kotlin",
   "blurb": "Kotlin is a multi-paradigm cross-platform, statically typed, general-purpose programming language for JVM, JS, Android/iOS and Native (via LLVM) targets.",
   "exercises": {
-    "concept": [],
+    "concept": [
+    ],
```

Comment:
this format is not a big deal, but looks strange since it's the only change in the file

Reply:
The thing is that this file is re-generated by a loot that uses moshi json exporter. :)